.idea/egi-marketplace.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is RubyMine project file. It needs to be ignored in .gitignore file. I suggest to add whole /.idea/ folder.